results/code-size.tex

```diff
@@ -155,6 +164,37 @@ \subsection{Executables}
 
 We see from \autoref{tab:size:c-vs-rust} that both the Circle Game and the Minimal Main  {\rust} binaries come quite close to the {\C} binaries in size.
 
+To see which protions of the {\rust} code that make the binary size increase, we now present a break down of the Sensor Tracker builds optimized for size.
+
+\begin{table}[H]
+  \centering
+  \begin{tabular}{l|r|r|l}
+    \textbf{Section}      & \textbf{C (B)} & \textbf{Rust (B)} & \textbf{Relative} \\
+    \hline
+    \textbf{app}          & 1776 & 3964 & 2.23x \\
+    \textbf{binding}      & 0    & 184  & N/A  \\
+    \textbf{emlib}        & 4348 & 4376 & 1.01x \\
+    \textbf{REL}          & 0    & 3516 & N/A  \\
+    \textbf{newlib}       & 2372 & 792  & 0.33x \\
+    \textbf{system}       & 460  & 896  & 1.95x \\
+    \textbf{unwind}       & 0    & 3820 & N/A  \\
+    \hline
+  \end{tabular}
+  \caption{}
+  \label{tab:size:breakdown}
+\end{table}
+
+In \autoref{tab:size:breakdown} we see that there are three major non-constant contributors\footnote{The \textbf{system} section increased with $\sim$2x, but this section will not increase as the complexity of the application is increased.} to the increased size for the {\rust} binary:
```

Comment:
Forslag til endring i fotnote (bytte ut "increase" med "remain constant"), f.eks.
"The system section increased with $\sim$2x, but the size of this section will remain constant while the complexity of the application increases"